docs/v1/get-achievement-of-the-week.md

```diff
@@ -131,6 +132,7 @@ if (response is NetworkResponse.Success) {
     {
       "user": "Tirbaba2",
       "rAPoints": "72",
+      "rASoftcorePoints": "150",
```

Comment:
I noticed that AotW has 'rAPoints' and achievementUnlocks has 'raPoints.' Should these match?

Reply:
Thanks for pointing that out - I will double check when I do some more maintenance on api-js. It's probably fine for now.